### PR TITLE
Add timestamp and additional unit tests

### PR DESCRIPTION
**Pull request by insacuri**
## What? and Why?
https://trello.com/c/HGU9OpOe/2173-add-dates-to-metadata-for-downstream-systems

started_at is being added to the eq data payload and the validator needs to be able to handle it

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,4 +4,4 @@ build:
 test:
 	pip3 install -r test_requirements.txt
 	flake8 --exclude lib
-	pytest -v --cov server --html=report.html
+	pytest -v --cov-report term-missing --cov=server --html=report.html
```
Comment (benhowes) [inline Makefile]: I always add `--cov-report term-missing` locally, but I don't normally commit. Is it okay to add it in?

File: tests/test_service.py [hunk 2]
```diff
@@ -7,6 +8,7 @@
 class TestValidateService(unittest.TestCase):
 
     validate_endpoint = "/validate"
+    healthcheck_endpoint = "/healthcheck"
```
Comment (benhowes) [inline tests/test_service.py:11]: Would suggest testing the health check in a separate class `TestHealthcheck` or similar.

**Review by benhowes — DISMISSED** [review]

**Comment by benhowes** [comment]
I've approved this because neither of the points I've raised are functional problems, but you may want to consider them.

File: Makefile (hunk 1 above)
Comment (insacuri) [reply Makefile]: Why would it not be?  Genuine question because it looks reasonable, but could have an effect that I don't know about

Comment (benhowes) [reply Makefile]: It's just that it prints a load of extra output at the end, which can be annoying when you're looking through test failures

Comment (insacuri) [reply Makefile]: That's fair, I'll remove it.  Would rather travis's output be clean and useful

**Review by benhowes — APPROVED** [review]

**Review by MebinAbraham — APPROVED** [review]
Working as expected, validates regardless of whether `started_at` is present or not. @insacuri Needs squashing.

**Pull request merged**